### DISCUSSION TITLE
Code-server settings and extensions can not be synchronized to VS code via 'setting sync'

### DISCUSSION THREAD

**Discussion by patialo** [coder/code-server#5420] [Q&A]
System: synology docker
code-server version: 4.4.0(And I can not upgrande to the lastet version , page would be blank when i install v4.5.1)

Actually， none of the sync extensions work for me
---
1.Use setting sync: Nothing was synced
![image](https://user-images.githubusercontent.com/31113826/183230809-45c9e01e-dc88-4d62-a170-51f3faac5827.png)
![image](https://user-images.githubusercontent.com/31113826/183230812-c9a328ff-0391-45f9-8e19-a1847e6807dc.png)
2.Use syncing: Nothing was synced
![image](https://user-images.githubusercontent.com/31113826/183231227-2f93f209-67fa-4e4f-b567-136b77738e72.png)
![image](https://user-images.githubusercontent.com/31113826/183231199-afd19703-c81d-4c8f-9855-bf4a82571e23.png)

This problem occurred after I upgraded to 4.5.1 with a blank page and then rolled back to 4.4.0

Does any setting file was changed during update？and how can I fix this？Give me some advice,plese

**Comment by patialo** [comment]
I tried to synchronize the settings of the old container to the newly created container, but still prompted that no extensons and settings were synchronized (code-server to code-server)

1.Upload in old container
![image](https://user-images.githubusercontent.com/31113826/183233740-61968a83-76a4-4489-ae7b-a6529af7953e.png)
2.Download in new container
![image](https://user-images.githubusercontent.com/31113826/183233810-c7b336dc-c3a8-4551-88ba-24591b0d24f4.png)

I'm definitely using the same extension market， and have configured the same token and GistID

**Reply by jsjoeio** [reply]
I think this is a known issue. From what I know, Settings Sync can't be used with code-server.

**Reply by patialo** [reply]
> I think this is a known issue. From what I know, Settings Sync can't be used with code-server.

I had resolved this problem ，and syncing well now. 
If there are similar issues or discussions, Maybe you can show them this.  @jsjoeio 


this is a bug in setting-sync。Settingsync synchronizes some unnecessary data, causing the gist to“Crash” with too much code

Here's the solution: 

Insert the following code into synclocalsettings.json
```
"ignoreUploadFolders": [
    "node_modules",
    "History",
    "workspaceStorage"
],
```
   https://github.com/shanalikhan/code-settings-sync/issues/1039

**Reply by jsjoeio** [reply]
Ah, I see. See these:
- https://github.com/coder/code-server/issues/3418
- https://github.com/coder/code-server/issues/2195
